Tests/myAccountTests.js

```diff
@@ -0,0 +1,97 @@
+var myAccountPage = require("../Pages/myAccountPage"),
+    homePage = require("../Pages/homePage"),
+    loginPage = require('../Pages/loginPage'),
+    data = require("../Data/data.js"),
+    UITasks = require("../Pages/UITasks"),
+    itemPage = require("../Pages/itemPage");
+    
```

Comment:
<img width="788" alt="Screenshot 2020-12-06 at 18 33 29" src="https://user-images.githubusercontent.com/50828644/101287646-ce235480-37f1-11eb-95cb-53766485cd09.png">
<img width="780" alt="Screenshot 2020-12-06 at 18 33 24" src="https://user-images.githubusercontent.com/50828644/101287648-cf548180-37f1-11eb-9447-cf0d03ad8139.png">

There are some errors shown when executing this test suite

Reply:
I added stability to the tests

Reply:
new error:
<img width="1286" alt="Screenshot 2020-12-08 at 09 59 52" src="https://user-images.githubusercontent.com/50828644/101462155-25cdd700-393c-11eb-90c3-8ab510f6cf9e.png">

Reply:
<img width="1286" alt="Screenshot 2020-12-08 at 09 59 52" src="https://user-images.githubusercontent.com/50828644/101462810-ec499b80-393c-11eb-88f0-64e0ac67c9a4.png">
Next run was successful

Reply:
Next run was successful
<img width="436" alt="Screenshot 2020-12-08 at 10 06 02" src="https://user-images.githubusercontent.com/50828644/101462867-008d9880-393d-11eb-95b0-e6aa060426cb.png">